blockchain/types/tx_internal_data_fee_delegated_smart_contract_deploy.go

```diff
@@ -307,7 +308,7 @@ func (t *TxInternalDataFeeDelegatedSmartContractDeploy) IntrinsicGas(currentBloc
 		gas += params.TxGasHumanReadable
 	}
 
-	gasPayloadWithGas, err := IntrinsicGasPayload(gas, t.Payload)
+	gasPayloadWithGas, err := IntrinsicGasPayload(gas, t.Payload, t.GetRecipient() == nil, *fork.Rules(big.NewInt(int64(currentBlockNumber))))
```

Comment:
I think we can explicitly input `true` or `false` instead of `t.GetRecipient() == nil` in cases of types transactions. Is there any reason for the equation?

Reply:
There's not specific reason for this equation. I tried to use the same code in IntrinsicGasPayload. However, explicitly setting `true` seems better. Applied at https://github.com/klaytn/klaytn/pull/1888/commits/76a49dc8b8b2a241df27e05edb95f9e04eb8ecdf.

Reply:
The changes [9ae1580 ](https://github.com/klaytn/klaytn/commit/9ae15809d565e179b9e3f723bf6c101c60977bc5) is safe? They call `vm.Call()`, then I think they guarantee the payload does not contain `create` or `create2` opcodes. Is it guaranteed?

Reply:
This change is about setting paramter of IntrinsicGas as 'true' or 'false'. IntrinsicGas function calculates the gas but it is done outside of vm. The `opcode` has no relation with IntrinsicGas. I'm not sure this is an answer to your question. Can you share what you referred?

Reply:
Sorry, Confused. That's correct. 

Did you intentionally leave? https://github.com/klaytn/klaytn/pull/1888/files#diff-436d633e7f524dca0da75c3bcf6322f3414bf7e9888e4e2b740d37c52077e6d0

Reply:
It's my fault. I thought deployment could possible with execution tx types. Fixed it.